web/base_viewer.js

```diff
@@ -1296,9 +1293,6 @@ class BaseViewer {
       return false;
     }
     const pageView = this._pages[pageNumber - 1];
-    if (!pageView) {
-      return false;
-    }
```

Comment:
*Slightly unrelated to the rest of the patch, but:* Given the existing validation just above it shouldn't be possible for `pageView` to be undefined here, and in any case passing `undefined` to the `PDFPageViewBuffer.has`-method won't break anything.